### DISCUSSION TITLE
en/latest/UserGuide/%E8%BD%A8%E8%BF%B9%E5%A4%84%E7%90%86/

### DISCUSSION THREAD

**Discussion by giscus** [zdsjjtTLG/TrackIt#15] [General]
# en/latest/UserGuide/%E8%BD%A8%E8%BF%B9%E5%A4%84%E7%90%86/

在python中轻松进行路径匹配

https://gotrackit.readthedocs.io/en/latest/UserGuide/%E8%BD%A8%E8%BF%B9%E5%A4%84%E7%90%86/

**Comment by Murphy666-code** [comment]
一般是先进行行程切分还是先轨迹数据清洗？

**Reply by zdsjjtTLG** [reply]
先行程切分

**Comment by hongjiala** [comment]
请问，如果GPS数据很大，清洗对应的速度是对应的线性增长吗

**Reply by zdsjjtTLG** [reply]
是的

**Comment by Hua0925-max** [comment]
请问为什么使用完停留点删除后，输出的可视化html文件大部分打开都是只有一个蓝色的点呢

**Reply by zdsjjtTLG** [reply]
你是轨迹预处理后输出的HTML图？那说明你的停留点删除的参数可能不合理，把所有的定位点都识别为停留点了。你好好看看停留点识别的参数含义

**Comment by L-uther** [comment]
您好，看了下源代码里的注释，trip_segmentations函数是根据相邻两点的时间差划分主行程。是否可以理解为，是根据车辆定位GPS的关闭时长来划分主行程。如果车辆的GPS在停留时未关闭，是不是需要先剔除部分速度为0的GPS点后，再用trip_segmentations函数划分行程呢
